fontc/src/workload.rs

```diff
@@ -232,11 +232,15 @@ impl<'a> Workload<'a> {
             .queued()
             .run();
 
-        let launchable: Vec<_> = self
+        let mut has_kern = false;
```

Comment:
to single out kerning in this context feels to me a bit out of place/too specific, ideally it'd be preferable to have a more generic way to prioritise tasks that does not involve special-casing